contrib/localnet/scripts/start-zetaclientd-genesis.sh

```diff
@@ -25,7 +25,7 @@ then
     rm ~/.tss/*
     MYIP=$(/sbin/ip -o -4 addr list eth0 | awk '{print $4}' | cut -d/ -f1)
     zetaclientd init  --zetacore-url zetacore0 --chain-id athens_101-1 --operator "$operatorAddress" --log-format=text --public-ip "$MYIP" --keyring-backend "$BACKEND"
-    zetaclientd start
+    zetaclientd start < /root/password.file
```

Comment:
This is required to start ZetaClient? We should add the change in `start-zetaclientd-background.sh` as well then since used for the upgrade test.